nupic/data/jsonhelpers.py

```diff
@@ -78,7 +80,7 @@ def validate(value, **kwds):
           ValidationError when value fails json validation
   """
 
-  assert len(kwds.keys()) >= 1
+  assert len(list(kwds.keys())) >= 1
```

Comment:
This kind of conversion would have been more efficient as `assert kwds` or `assert len(kwds)` or `assert len(kwds) >=1`